src/main/java/net/greatstart/dao/hibernate/UserDaoImpl.java

```diff
@@ -2,22 +2,28 @@
 
 import net.greatstart.dao.UserDao;
 import net.greatstart.model.User;
+import org.hibernate.HibernateException;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.hibernate.HibernateException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)